### DISCUSSION TITLE
How close is the clipboard integration to being done?

### DISCUSSION THREAD

**Discussion by MonstrousOgre** [jeffreytse/zsh-vi-mode#264] [General]
I was just curious. The Readme has said that it's in progress for a really long time. I would love to be able to use it with the clipboard.

**Comment by einarpersson** [comment]
1.5 year later, wondering the same thing!

**Comment by anderslundstedt** [comment]
FWIW there are some workarounds that seem popular:

- One of mine: https://github.com/jeffreytse/zsh-vi-mode/issues/19#issuecomment-1268057812
- Another popular one: https://github.com/jeffreytse/zsh-vi-mode/issues/19#issuecomment-1915625381

**Comment by jeffreytse** [comment]
Thanks for your patience, now the system clipboard integration has been supported, please check out the latest version. : )
